xml/System.Net/WebHeaderCollection.xml

```diff
@@ -1482,13 +1482,16 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="name" Type="System.String" Index="0" FrameworkAlternate="dotnet-uwp-10.0;net-5.0;netcore-1.0;netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netstandard-2.1" />
+        <Parameter Name="name" Type="System.String" Index="0" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <param name="name">Header name.</param>
+        <summary>Gets or sets the specified response header.</summary>
+        <value>A <see cref="T:System.String" /> instance containing the specified header.</value>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentOutOfRangeException">The length of <paramref name="value" /> is greater than 65535.</exception>
+        <exception cref="T:System.InvalidOperationException">This <see cref="T:System.Net.WebHeaderCollection" /> instance does not allow instances of <see cref="T:System.Net.HttpResponseHeader" />.</exception>
```

Comment:
Looks like copy-paste error

Reply:
Do you mean the typo you suggested to fix or is there something which shouldn't be there?

Reply:
I meant this specific exception -- what does it mean for this indexer to not allow instances of `HttpResponseHeader`?

Reply:
I removed this property completely. More details are here: https://github.com/dotnet/dotnet-api-docs/pull/4268#discussion_r438025973

Reply:
@karelz, are you fine with the resolution? PR is blocked by your requested change.